Ginger/GingerCoreCommon/Repository/RepositoryFolder.cs

```diff
@@ -82,6 +82,11 @@ public string DisplayName
             }
         }
 
```

Comment:
Do it on RepositoryFolderBase

Reply:
not possible for this (Folder items ) to move to the FolderBase Meni. I'm keeping this for now.